ext/afform/core/afform.php

```diff
@@ -466,6 +466,17 @@ function afform_civicrm_permissionList(&$permissions) {
   }
 }
 
+/**
+ * Implements hook_civicrm_alterApiRoutePermissions().
+ *
+ * @see CRM_Utils_Hook::alterApiRoutePermissions
+ */
+function afform_civicrm_alterApiRoutePermissions(&$permissions, $entity, $action) {
+  if ($entity == 'Afform' && $action == 'submit') {
+    $permissions = CRM_Core_Permission::ALWAYS_ALLOW_PERMISSION;
```

Comment:
The contract for `CRM_Utils_Hook::alterApiRoutePermissions` specifies that `$permissions` is an alterable array. This assigns it as a singular string. Seems like one side or the other should change.